src/redis.c

```diff
@@ -3006,10 +3009,22 @@ sds genRedisInfoString(char *section) {
             struct redisCommand *c = redisCommandTable+j;
 
             if (!c->calls) continue;
+
+            sample_t percentiles[CMDTIME_PERCENTILES_COUNT];
+            calculatePercentiles(
+                &c->samples, CMDTIME_PERCENTILES_COUNT, CMDTIME_PERCENTILES, percentiles);
+
             info = sdscatprintf(info,
-                "cmdstat_%s:calls=%lld,usec=%lld,usec_per_call=%.2f\r\n",
+                "cmdstat_%s:calls=%lld,usec=%lld,usec_per_call=%.2f",
                 c->name, c->calls, c->microseconds,
                 (c->calls == 0) ? 0 : ((float)c->microseconds/c->calls));
+            for (int i = 0; i < CMDTIME_PERCENTILES_COUNT; i++) {
```

Comment:
nit: I think the c/c++ style was the use pre-increment. Something about post increment requiring the value to be stored, if the compiler doesn't recognize that it's not used.